src/app/body/milestone-details/milestone-details.component.ts

```diff
@@ -62,15 +62,19 @@ export class MilestoneDetailsComponent implements OnInit {
   appkey: string = "";
   addTaskActive: boolean = true;
   editMilestoneActive: boolean = false;
+  tasks: Tasks [] =[];
 
   public tasksObservable: Observable<Tasks[]>;
   public milestoneObservable: Observable<Milestones[]>
   constructor(private route: ActivatedRoute, private router: Router, private functions: AngularFireFunctions, public startService: StartServiceService, public backendService: BackendService, public navbarHandler: NavbarHandlerService, private location: Location, public authService: AuthService, public errorHandlerService: ErrorHandlerService, public toolsService: ToolsService, public popupHandlerService: PopupHandlerService, public applicationSetting: ApplicationSettingsService, public validationService: ValidationService, public milestoneService: MilestoneServiceService) {
   }
 
   ngOnInit(): void {
+    console.log("check1",this.milestoneService.taskData); //To Test in Production
+    this.tasks = [];
     this.milestoneId = this.route.snapshot.params['MilestoneId'];
     this.navbarHandler.addToNavbar(this.milestoneId);
+    console.log(this.milestoneService.taskData);
```

Comment:
remove console

Reply:
Sanjay bhaiya said, its need to test in production.